### PR TITLE
VACMS-4203: remove computed field module

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #4203 - this is a follow-up PR to #4364 and should not be merged until it has been deployed.

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_govdelivery/src/EventSubscriber/EntityEventSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  public function __construct(ProcessStatusBulletin $process_status_bulletin) {
+    $this->processStatusBulletin = $process_status_bulletin;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() : array {
+    return [
+      HookEventDispatcherInterface::ENTITY_INSERT => 'handleEntityUpsert',
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_govdelivery/src/EventSubscriber/EntityEventSubscriber.php:39]: bonus points for great new word.

**Review by swirtSJW — APPROVED** [review]
Overall this looks amazing.  I am  a little hesitant about this PR also tackling the @todo of looking to see if it was ever published.   
That  needs some hand testing steps to make sure it is sending when expected and not when not.  I think it is likely fine... but we are lumping in a lot of improvements into one here.

File: docroot/modules/custom/va_gov_govdelivery/src/EventSubscriber/EntityEventSubscriber.php (hunk 1 above)
Comment (acrollet) [reply docroot/modules/custom/va_gov_govdelivery/src/EventSubscriber/EntityEventSubscriber.php:39]: @swirtSJW I wish I could take credit :-D

https://wiki.postgresql.org/wiki/UPSERT

**Comment by acrollet** [comment]
Closing in favor of #4451

**Pull request closed without merging**